cost-analyzer/templates/kubecost-cluster-controller-actions-config.yaml

```diff
@@ -38,4 +38,18 @@ metadata:
 binaryData:
   config: |
 {{- toJson .Values.clusterController.actionConfigs.containerRightsize | b64enc | nindent 4 }}
+{{- end }}
+
+{{- range .Values.clusterController.actionConfigs.clusterTurndown }}
+---
+apiVersion: kubecost.com/v1alpha1
+kind: TurndownSchedule
+metadata:
+  name: {{ .name }}
+  finalizers:
+  - "finalizer.kubecost.com"
+spec:
+  start: {{ .start }}
+  end: {{ .end }}
+  repeat: {{ .repeat }}
```

Comment:
Have you tested this to ensure the CR is properly created? The CRD is deployed as part of the `kubecost-cluster-controller-template.yaml` template.

Reply:
I ran it through helm template --validate. Was figuring we can rely on helm to install CRD before the CRs that use them

Reply:
Can you do an actual installation to ensure that happens? Generally speaking, it's not a great idea to start mixing standard and custom resources in the same template file.

Reply:
do we have a separate ci-values.yaml file? would be nice to deploy cluster controller to test stuff like this in general in the kind matrix

Reply:
Yes, it's being done [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/ci/statefulsets-cc.yaml#L22-L23) already. Action run for it is [here](https://github.com/kubecost/cost-analyzer-helm-chart/actions/runs/7872700021/job/21478482827#step:11:171). I guess it's fine.

Reply:
we can add an enhancement to those CI values that instantiate that pre-defined config just to confirm this case is true. I will hold the action item to add something like that

Reply:
Would you be logging a Jira issue or a GitHub issue? It would indeed be nice to supply some values here and ensure they are laid down by the CR.

Reply:
I was just going to do it, perhaps in like 30-60 mins?

Reply:
I would still create a GitHub enhancement to track.